sdk/storage/azure-storage-blob/azure/storage/blob/_shared/models.py

```diff
@@ -487,3 +488,41 @@ def __init__(self):
         self.signed_service = None
         self.signed_version = None
         self.value = None
+
+class TokenAudience():
+    """
+    Base class that defines what calling functions and attributes should be defined for an Azure Active
+    Directory audience token.
+
+    :param str value: The audience to use when requesting tokens for Azure Active Directory authentication.
+        For more information please take a look at:
+        https://learn.microsoft.com/azure/storage/blobs/authorize-access-azure-active-directory. Please use one
+        of the provided classmethods over creating a custom value unless you have a specific scenario for doing so.
+    """
+    DEFAULT_OAUTH_SCOPE = ".default"
+    STORAGE_OAUTH_SCOPE = "https://storage.azure.com/"
```

Comment:
These should probably be private since this class is exported? (Add `_` in front)